templates/BoaJava.stg

```diff
@@ -126,8 +126,10 @@ new <type>()
 >>
 
 TupleType(name, fields, types, protos) ::= <<
-private class <name>
-{
+private class <name> implements boa.runtime.Tuple, java.io.Serializable
+{	
+	<name>(){}
```

Comment:
is it necessary? what would an 'empty' tuple even be set to? where are we making empty tuples?

Reply:
Yes, I have created a "load" function to load a trained model, which uses empty tuples. For example, 
___model_5 = (boa.types.ml.BoaLinearRegression)boa.functions.BoaIntrinsics.load("/home/tess/Desktop/datagen/0/train1587614590388ML.model", new BoaTup_0());